spec/factories/course_assessment_questions.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :course_assessment_question, class: Course::Assessment::Question do
-    assessment
```

Comment:
build_stubbed would call the callback I think. so the course don't have a root_folder and there are exceptions.